bot/exts/utilities/reddit.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, bot: Bot):
         self.access_token = None
         self.client_auth = BasicAuth(RedditConfig.client_id.get_secret_value(), RedditConfig.secret.get_secret_value())
 
-        self.auto_poster_loop.start()
+        # self.auto_poster_loop.start()
```

Comment:
Since we're not starting the loop, do we also need to comment out the `self.auto_poster_loop.cancel()` call in `cog_unload()` (and anywhere else)?

Reply:
No, cancel doesn't error if the task isn't running.